charts/lnd/charts/thunderhub/templates/deployment.yaml

```diff
@@ -0,0 +1,82 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "thunderhub.fullname" . }}
+  labels:
+    {{- include "thunderhub.labels" . | nindent 4 }}
+spec:
+  selector:
+    matchLabels:
+      {{- include "thunderhub.selectorLabels" . | nindent 6 }}
+  template:
+    metadata:
+      annotations:
+      {{- with .Values.podAnnotations }}
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      labels:
+        {{- include "thunderhub.selectorLabels" . | nindent 8 }}
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      serviceAccountName: {{ include "thunderhub.serviceAccountName" . }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      containers:
+        - name: {{ .Chart.Name }}
+          securityContext:
+            {{- toYaml .Values.securityContext | nindent 12 }}
+          image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
+          imagePullPolicy: {{ .Values.image.pullPolicy }}
+          env:
+          - name: ACCOUNT_CONFIG_PATH
+            value: "/home/thunderhub.yaml"
+          - name: LOG_LEVEL
+            value: 'debug'
+          - name: YML_ENV_1
```

Comment:
we could use a better name

Reply:
Thunderhub requires us to use this env var name.

Reply:
ok